tests/features/iocommands.feature

```diff
@@ -0,0 +1,11 @@
+Feature: I/O commands
+
+  @wip
```

Comment:
With behave library, you can add any tags to your tests, to run just a subset of them. For example, if I mark one scenario with the tag "@foo", and then run "behave --tags=foo", only this scenario will be run.

The "wip" tag is special in behave, because this is how they chose to denote "work in progress". If you mark your feature "@wip", you can run it with as usual, with "behave --tags=wip". Or, you can use a shortcut: "behave -w". In addition to just running one feature, the "-w" switch will switch off stderr and stdout capture, so you'll see all that in the console (normally it's all hidden), and will add a flag to stop on any error.

I got into the habit of marking the last feature I worked  on as "@wip", because this way I can quickly run it separately. It does not matter for travis, because it always runs "behave" on the whole suite.